modules/ecs/cluster/main.tf

```diff
@@ -1,26 +1,35 @@
+locals {
+  current_account_id = data.aws_caller_identity.current.account_id
+}
+
+data "aws_caller_identity" "current" {}
+
 resource "aws_ecs_cluster" "ecs_cluster" {
   name = "nerves-hub-${var.environment}"
 
   lifecycle {
     create_before_destroy = true
   }
-}
 
-resource "aws_iam_role_policy" "ecs_instance_role_policy" {
-  name   = "ecs-instance-role-policy-${var.environment}"
-  policy = file("${path.module}/templates/ecs-instance-role-policy.json")
-  role   = aws_iam_role.ecs_role.id
+  tags = var.tags
 }
 
-resource "aws_iam_role" "ecs_role" {
-  name               = "ecs-instance-role-${var.environment}"
-  assume_role_policy = file("${path.module}/templates/ecs-role.json")
+resource "aws_iam_role" "ecs_runtime" {
```

Comment:
So from what I can tell this policy (before and after your changes) isn't being used anywhere. 
This setup is a little different than our applications. The cluster is shared between different services and each service has their own sets of secrets they need access to, which is why each service has their own iam role to assume for runtime rather than one for the whole cluster.

Reply:
Yeah that makes sense. Make each service declare it's permissions